packages/flutter_tools/lib/src/flx.dart

```diff
@@ -283,7 +288,8 @@ Future<int> assemble({
 
   zipBuilder.addEntry(_createAssetManifest(assets));
 
-  ZipEntry fontManifest = _createFontManifest(manifestDescriptor, usesMaterialDesign ? _getMaterialFonts() : null);
+  ZipEntry fontManifest = _createFontManifest(manifestDescriptor,
+      usesMaterialDesign ? _materialFontsManifest : null);
```

Comment:
We also want to make this conditional on whether we're building an FLX for iOS or for Android.  On Android, these fonts are already on the system so we don't need to pay the cost of including them in the FLX.